frontend/templates/contraceptives/tabs/Overview.tsx

```diff
@@ -0,0 +1,82 @@
+import { ReactElement } from 'react';
+import styled from 'styled-components';
+import PillRow from '../../../components/PillRow';
+
+// Styling
+const CategoryRow = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin: 2rem 0;
+  width: 100%;
+`;
+
+const CategoryTitle = styled.h3`
+  color: #bebebe;
+  font-size: 1rem;
+  font-weight: normal;
+  margin: 0;
+`;
+
+const CategoryValue = styled.p`
+  color: #4b4b4b;
+  font-size: 1.5rem;
+  font-weight: bold;
+  margin: 0;
+  white-space: nowrap;
+`;
+
+const Column = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const Description = styled.p`
+  color: #7c7c7c;
+`;
+
+// Components
+const Category = ({
+  title,
+  value,
+}: {
+  title: string;
+  value: string;
+}): ReactElement => {
+  return (
+    <Column>
```

Comment:
We probably want to add some margin for each Category in the Category row. Right now, I'm not getting any wrapping for the elements in the row, so it looks kind of squished. 

Also, we probably want to make the CategoryValue less bold


<img width="501" alt="Screen Shot 2021-11-07 at 7 15 55 PM" src="https://user-images.githubusercontent.com/17476012/140667530-fa0fbf5a-5a1f-4381-b8b0-2c02d34f80a5.png">